runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -442,6 +444,20 @@ TR_RelocationRuntime::relocateAOTCodeAndData(U_8 *tempDataStart,
 
    reloLogger()->relocationDump();
 
+   if (_comp->isRemoteCompilation())
+      {
+      if (!_comp->compileRelocatableCode() || TR::CompilationInfo::canRelocateMethod(_comp))
+         {
+         fej9()->_compInfoPT->relocateThunks();
+         }
+      else
+         {
+         // metaData == NULL indicates AOT relocation is delayed
```

Comment:
This comment refers to metadata, but there is no such thing in here.
Also, in cases when we delay AOT relocations of the AOT code we just produced, does `relocateAOTCodeAndData` get called at all?

Reply:
Good point about delayed relocations. Will have to think about it more.

Reply:
Ok, so `relocateAOTCodeAndData` is not called if relocations are delayed, thus thunks will not get stored to SCC when they should.
I could move the AOT part of the code to its previous location, and leave the non-AOT part here, because `relocateThunks` needs to be called before the rest of the relocations are done.
Would make code even more confusing though.